NILICENSE

```diff
@@ -0,0 +1,4 @@
+
+Usage of GrpcSessionOptions on session initialization are subject to the NI General Purpose EULA, available at ni.com
```

Comment:
Instead of "Usage of GrpcSessionOptions on session initialization" (ambiguous, paraphrasing) consider using precise terminology, like:

```
Passing a GrpcSessionOptions instance as a paremeter to Session.__init__() is subket to ....
```

Reply:
Yeah, sounds good. Let me update.